quic/s2n-quic-transport/src/space/application.rs

```diff
@@ -440,6 +445,22 @@ impl<Config: endpoint::Config> ApplicationSpace<Config> {
         )
     }
 
+    /// Returns `true` if sending is limited by the application and not the congestion controller
+    ///
+    /// Sending is app limited if the application is not fully utilizing the available
+    /// congestion window currently and there is no more application data remaining to send.
+    fn is_app_limited(&self, path: &Path<Config>, bytes_sent: usize) -> bool {
+        let cwnd = path.congestion_controller.congestion_window();
```

Comment:
Would it be worth moving all of this to `Path` in something like `is_congestio_blocked`?